kie-wb/kie-wb-webapp/pom.xml

```diff
@@ -1504,7 +1502,7 @@
             <server>org.jboss.errai.cdi.server.gwt.EmbeddedWildFlyLauncher</server>
             <disableCastChecking>true</disableCastChecking>
             <runTarget>kie-wb.html</runTarget>
-            <hostedWebapp>src/main/webapp</hostedWebapp>
+            <hostedWebapp>${project.build.directory}/kie-wb.war</hostedWebapp>
```

Comment:
We should be able to configure the maven in way that all resources from src/main/webapp are copied into target/something. So basically the result would be the same, but there would be less configuration in the pom.